guide/plugins_apollocar3d.ipynb

```diff
@@ -210,14 +176,14 @@
     "\n",
     "## Everything else\n",
     "All pifpaf options and commands still stand, read more in the\n",
-    "[OpenPifPaf guide](https://openpifpaf.github.io/openpifpaf/intro.html). If you are interested in training your own dataset, read the section on a [Custom Plugin](https://openpifpaf.github.io/openpifpaf/plugins_custom.html)."
+    "[OpenPifPaf guide](https://openpifpaf.github.io/openpifpaf/intro.html). If you are interested in training your own dataset, read the section on a {doc}`custom dataset <plugins_custom>`."
```

Comment:
Regarding the first link: This is now a page of the Guide and I find it a bit odd to link to the Guide so explicitly here. In any case, this link is also broken.

Reply:
Thanks, I agree. I have just removed the link.